frontend/src/scenes/insights/EmptyStates.tsx

```diff
@@ -70,15 +70,15 @@ export function TimeOut({ isLoading }: { isLoading: boolean }): JSX.Element {
                             rel="noopener"
                             target="_blank"
                         >
-                            Upgrade PostHog
+                            Upgrade PostHog to Enterprise Edition
                         </a>{' '}
-                        to enterprise to get access to Clickhouse
+                        and get access to a superscalable backend based on the ClickHouse database.
```

Comment:
Nit: not sure about "superscalable", doesn't sound very serious

Reply:
Possibly, do you have different wording in mind?

Reply:
Hmmm how about "[...] get access to a backend engineered for high scalability" or "[...] get access to a backend engineering for billion+ scalability"